lib/faraday/adapter.rb

```diff
@@ -35,7 +35,7 @@ def inherited(subclass)
     extend Parallelism
     self.supports_parallel = false
 
-    def initialize(app = nil, opts = {}, &block)
```

Comment:
This is weird. Why is that argument unused???

Reply:
Adapters have now become endpoints for the stack, they are always put at the end of it.
So `app` becomes useless. However, to ensure backwards-compatibility, we need to accept it

Reply:
This is perhaps worth a documentation comment (YARD format comment for the parameters).